recipes/libzip/all/conanfile.py

```diff
@@ -121,20 +130,21 @@ def package_info(self):
         self.cpp_info.components["_libzip"].libs = ["zip"]
         if self.settings.os == "Windows":
             self.cpp_info.components["_libzip"].system_libs = ["advapi32"]
-            if self._crypto == "win32":
+            if self.options.crypto == "win32":
                 self.cpp_info.components["_libzip"].system_libs.append("bcrypt")
         self.cpp_info.components["_libzip"].requires = ["zlib::zlib"]
         if self.options.with_bzip2:
             self.cpp_info.components["_libzip"].requires.append("bzip2::bzip2")
         if self.options.with_lzma:
             self.cpp_info.components["_libzip"].requires.append("xz_utils::xz_utils")
-        if self.options.with_zstd:
+        if self.options.get_safe("with_zstd"):
             self.cpp_info.components["_libzip"].requires.append("zstd::zstd")
-        if self._crypto == "openssl":
+        if self.options.crypto == "openssl":
             self.cpp_info.components["_libzip"].requires.append("openssl::openssl")
```

Comment:
Does the `crypto` comopnent suffice?
```suggestion
            self.cpp_info.components["_libzip"].requires.append("openssl::crypto")
```

Reply:
It should be sufficient: https://github.com/nih-at/libzip/blob/6a0ea5711d7d7fa12c65a923439f589e575a5680/lib/CMakeLists.txt#L164